README.md

```diff
@@ -8,6 +8,9 @@ The goal of the project is to produce useful visualizations and statistics for O
 
 Meeting notes can be found in [this Google Doc](https://docs.google.com/document/d/11xji54-RiszyFBQnSBOI5Ylmzn2vC9glwAoU6A8CM_0/edit?pli=1#).
 
+## To install the backend in a Vagrant virtual box, follow the instructions here:
+[Instructions for running backend in vagrant](https://github.com/stochasticTreat/opendisclosure/blob/master/README_installation_in_vagrant.md)
```

Comment:
It'd be great to make this link point to the openoakland repo instead of yours. I think you can actually make it relative by doing a link like:

`[Instructions](blob/master/README_installation_in_vagrant.md)`